### ISSUE TITLE
Capitalization of X-Ray

### ISSUE THREAD

**Issue by retorquere** [zeke/ap-style-title-case#2]
ap-title-case title-cases `X-Ray` as `X-ray`; https://titlecaseconverter.com/ capitalizes to `X-Ray` in `AP` mode.

**Comment by zeke** [comment]
Based on your message above, it looks like this module and that website are both producing the same value. Am I missing something?

**Comment by retorquere** [comment]
Sorry, mistyped that, updated original message

**Comment by zeke** [comment]
Are you sure? Here's the output I'm seeing from the latest module:

```
$ npx trymodule ap-style-title-case

> ap_style_title_case('X-Ray')
'X-Ray'
```

**Comment by retorquere** [comment]
Jeez I'm a doofus, sorry. What I had meant to say is that this:

```
const titleCase = require('ap-style-title-case')
console.log(titleCase("Observations of isolated pulsars and disk-fed X-ray binaries."))
```

outputs `Observations of Isolated Pulsars and Disk-fed X-ray Binaries.`, where https://titlecaseconverter.com/ outputs `Observations of Isolated Pulsars and Disk-Fed X-Ray Binaries.`

**Comment by zeke** [comment]
I seeee! Thanks for enduring. I'd be open to changing the behavior for hyphenated words. Would you like to take a shot at submitting a pull request for this?

**Comment by retorquere** [comment]
Done in #3

**Comment by zeke** [comment]
Landed in `1.1.0` 🎉 

https://github.com/zeke/ap-style-title-case/pull/3#issuecomment-452783388
